scripts/provision.sh

```diff
@@ -2,8 +2,26 @@
 #
 # This script is run by Vagrant when a new machine is provisioned.
 #
+ 
+sudo apt-get -y install git
+# add support for 64 bit (which vagrant is set to configure)
+sudo apt-get -y install lib32z1 lib32ncurses5 lib32bz2-1.0
```

Comment:
don't you force 64 bit https://github.com/espruino/Espruino/blob/master/Vagrantfile#L7 ?